core/client/app/controllers/settings/general.js

```diff
@@ -76,6 +76,20 @@ export default Controller.extend(SettingsSaveMixin, {
         }
     }),
 
+    facebookUrl: computed('model.facebook', function () {
+        return this.get('model.facebook');
+    }),
+
+    twitterUrl: computed('model.twitter', function () {
```

Comment:
@kevinansfield do you think it would actually make more sense, to work with a transform here?